src/__docs__/migrating-from-react-use.story.mdx

```diff
@@ -507,9 +507,9 @@ Not implemented yet
 
 Not implemented yet
 
-#### useLogger
```

Comment:
Better to leave `useLogger` as it is migration guide.

Reply:
Done